Source/WebKit/GPUProcess/media/InitializationSegmentInfo.serialization.in

```diff
@@ -31,7 +31,7 @@ struct WebKit::InitializationSegmentInfo {
 
 [Nested] struct WebKit::InitializationSegmentInfo::TrackInformation {
     WebKit::MediaDescriptionInfo description;
-    WebKit::TrackPrivateRemoteIdentifier identifier;
+    uint64_t id;
```

Comment:
Ditto.

Reply:
this require to either create a TrackID.h or modify the python generator to use the header where it's defined..